pom.xml

```diff
@@ -1518,7 +1518,7 @@
           <plugin>
             <groupId>com.sun.tools.jxc.maven2</groupId>
             <artifactId>maven-jaxb-schemagen-plugin</artifactId>
-            <version>[1.2,2.0)</version>
```

Comment:
Version ranges are valid AFAIK, what's complaining about this?

Reply:
Might be an issue with IntelliJ's maven integration, unable to resolve dependency with range.

I searched the rest of the pom.xml files `grep -R --include=pom.xml maven-jaxb-schemagen-plugin -C1` and the ones using this plugin are hard set to 1.2.

I figured hard-setting this one to 1.2 like the others was minor enough to clear up an IDE. No worries either way.

Reply:
Thanks for the note. Using a fixed version is fine, was just curious on the motivation.